src/test/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationServiceTest.java

```diff
@@ -220,6 +220,63 @@ void shouldSubmitResponseToApplicationAndSendCopy() throws CaseDocumentException
         );
     }
 
+    @Test
```

Comment:
I can't see the unit test for when the upload file is included then createPdfOfResponse should be called.